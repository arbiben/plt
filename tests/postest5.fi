 include for loop and while loop here amongst other things $
