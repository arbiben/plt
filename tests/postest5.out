struct three {
int one?
int two?
char f?
}

struct four {
int fi?
}


bool running()
{
for (a = 0? a < 5 or F? a = a + 1) {
if (a > 6)
{
print(errormessage, nameofuser)?
}
}
for (i = 10? T? i = i - 10) {
if (i < 0 and i is i)
{
return i?
}
}
}

